lib/init.php

```diff
@@ -115,52 +124,55 @@ function beans_add_theme_support() {
 	// Beans specific.
 	add_theme_support( 'offcanvas-menu' );
 	add_theme_support( 'beans-default-styling' );
-
 }
 
 add_action( 'beans_init', 'beans_includes' );
 /**
  * Include framework files.
  *
+ * @since 1.0.0
  * @ignore
+ * @access private
```

Comment:
The access is not private.  Go ahead and remove the `@access` tag.  We do want to ignore it though for the automatic doc generator.